docs/custom_schemas/h5md/explanation-H5MD-NOMAD.md

```diff
@@ -0,0 +1,1342 @@
+
+* `indices`
+: a list of integer indices corresponding to all particles belonging to this group. Indices should correspond to the list of particles stored in the `particles/all` group.
+
+* `is_molecule`
+: indicator of individual molecules (typically with respect to the bond connections defined by a force field).
+
+* `custom_dataset`
+: arbitrary additional metadata for this particle group may be given.
+
+
+Each subgroup may also contain a (nested) `particles_group` subgroup, in order to subdivide the group of particles into an organizational hierarchy. As with the overall `particles_group` container, the groups contained within `particles_group` must not *partition* the particles within this group (i.e., overlapping or non-complete groupings are allowed). However, particle groups *must* contain particles already contained within the parent `particles_group` (i.e., subgroups must be a subset of the grouping at the previous level of the hierarchy).
+
+Note that typically the `particles_group` hierarchy ends at the level of individual particles (i.e., individual particles are not stored, since this information is already contained within the `particles` group).
+
+
+
+## The parameters group
+
+The initial H5MD proposed a simple and flexible schema for the storage of general "parameter" information within the `parameters` group, with the following structure:
+
+    parameters
+     +-- <user_attribute1>
+     \-- <user_data1>
+     \-- <user_group1>
+     |    \-- <user_data2>
+     |    \-- ...
+     \-- ...
+
+In contrast, the H5MD-NOMAD schema calls for very specific structures to be used when storing parameter information. While the previous groups have attempted to stay away from enforcing NOMAD-specific data structures on the user, instead opting for more intuitive and generally-convenient structures, the `parameters` group utilizes already-existing metadata and structures within NOMAD to efficiently import simulation parameters in a way that is searchable and comparable to simulations performed by other users.
+
+In this way, the H5MD-NOMAD `parameters` group has the following structure:
+
+    parameters
+     \-- <parameter_subgroup_1>
+     |    \-- ...
+     \-- <parameter_subgroup_2>
+     |    \-- ...
+     \-- ...
+
+The subgroups `force_calculations` and `workflow` are supported. The following describes the detailed data structures for these subgroups, using the NOMAD MetaInfo definitions for each underlying `Quantity`. Please note that:
```

Comment:
Maybe adding href to Glossary? If Quantity does not exist in Glossary, feel free to create it.

Reply:
added a TODO for now. I guess I would like to wait until I add this to the main docs and then develop these links there.